docs/web-component-samples/src/index.ts

```diff
@@ -1,6 +1,4 @@
-import store from '@webex/cc-store';
-import '@webex/cc-station-login';
-import '@webex/cc-user-state';
+import {store, WebUserState, WebStationLogin} from '@webex/cc-widgets';
```

Comment:
As discussed during sitdown review, let us implement this one,

```suggestion
import {store} from '@webex/cc-widgets';
import '@webex/cc-widgets/wc';
```

Reply:
Done